### PR TITLE
Add LR scheduler and change optimizer options

### PR DESCRIPTION
**Pull request by PhysSong**
Make the optimizer work as documented, except the [undocumented warmup steps](https://github.com/columbia-ai-robotics/dsr/blob/f34d60f885cd01e6b562e799d7c81eafda3ae765/train.py#L173-L174).

...And I noticed that they also used $5\cdot10^{−4}$ instead of $10^{−4}$ as the initial learning rate on fine-tuning, which is one more undocumented change. :scream:

### CODE REVIEW

**Review by yycho0108 — APPROVED** [review]
LGTM. It's unfortunate that DSR has so many undocumented parts.
